src/integration/bluesky.utils.ts

```diff
@@ -0,0 +1,38 @@
+import type { Actor } from '@/integration/bluesky-api.client';
+
+export const BRIDGY_FED_LABEL = 'bridged-from-bridgy-fed-activitypub';
+
+export function findValidBridgyHandle(
+    actors: Actor[],
+    domain: string,
+): string | null {
+    // Find Bridgy Fed actors
+    const bridgyActors = actors.filter((actor) => {
+        if (!actor.labels) return false;
+
+        return actor.labels.some((label) => label.val === BRIDGY_FED_LABEL);
+    });
+
+    // Find actors with a handle ending in .{domain}.ap.brid.gy
+    const expectedHandleSuffix = `.${domain}.ap.brid.gy`;
```

Comment:
question: I'm curious if this handles sites on www as expected?

For a site hosted on www, e.g. www.sambinder.com, `accounts.apId.hostname` is `www.sambinder.com`

However, as Bluesky Fed strips www (sometimes), the search results may return a handle without www, e.g. `art.sambinder.com.ap.brid.gy` - which won't match `.www.sambinder.com.ap.brid.gy` suffix?

https://public.api.bsky.app/xrpc/app.bsky.actor.searchActors?q=www.sambinder.com

Reply:
Yeh good catch, we need to normalize the domain first before it can be used to match on 👍